tempest/api/workloadmgr/base.py

```diff
@@ -969,6 +978,24 @@ def get_restored_volume_list(self, restore_id):
         LOG.debug("restored volume list:" + str(restored_volumes))
         return restored_volumes
 
+    '''
+    Method returns the list of restored volumes id an type
+    '''
+
+    def get_restored_volume_info_list(self, restore_id):
+        resp, body = self.wlm_client.client.get("/restores/" + restore_id)
```

Comment:
Instead of adding these API calls again, why not use get_restored_vm_list() and get_restored_volume_list() methods which are already available?

Reply:
updated function